### DISCUSSION TITLE
Where are the printer settings

### DISCUSSION THREAD

**Discussion by bdawg4113** [OrcaSlicer/OrcaSlicer#7810] [General]
![image](https://github.com/user-attachments/assets/dbfc1669-04d0-46c2-b81c-cfec7e37a29a)

On my prepare screen, I don't see the printer settings anywhere, did I download V2.1.1 incorrectly? 

I downloaded the exe file for V2.1.1 about a couple months ago and left orca alone for a couple months. Now that I'm finishing setting up my new bambu, I realize that I can't find the printer settings page. Where is it?

**Comment by LJKraus** [comment]
When the settings panel is minimized, a couple arrows _should_ be in the upper left corner, click there:
![image](https://github.com/user-attachments/assets/638186f0-6f00-42a1-b143-cbd950d25f49)

If that fails, "Reset Window Layout" in the View menu should bring the panel back.
